INSTALL.rst

```diff
@@ -50,13 +47,14 @@ Run the install script:
   
   python install.py
 
-If you successfully followed the instruction above cyclus binary have been
-generated and be placed in the  ``.local/`` in your home directory. 
-You need to had ``~/.local/bin`` to the bottom of your ``$PATH``:
+If you successfully followed the instruction above, the cyclus binary has been
+generated and placed in the  ``~/.local/`` in your home directory. 
+You need to add ``~/.local/bin`` to the front of your ``$PATH``:
```

Comment:
This initially referred to the "bottom" of the path, but the example appears to be the front (top?)